code/modules/reagents/chemistry/reagents/medicine.dm

```diff
@@ -886,7 +886,7 @@
 /datum/reagent/medicine/stimulative_agent
 	name = "Stimulative Agent"
 	id = "stimulative_agent"
-	description = "An illegal compound that dramatically enhances the body's performance and healing capabilities."
+	description = "Increases run speed and eliminates stuns, can heal minor damage. If overdosed it will deal toxin damage and stun."
```

Comment:
Nitpick. It will deal stamina damage Not stun. Yet the from the drug effect itself will out heal the stamina damage. But it will then be a bit less effective